### PR TITLE
Fix warnings and issues in docs

### PR DESCRIPTION
**Pull request by akantak**

### CODE REVIEW

File: docs/log_metric_callback.rst [hunk 1]
```diff
@@ -39,8 +39,8 @@ You could also add your custom metric logging function by defining the function
         your code 
 
 
-Example of a Metric Callback
-============================
+Example of a MLFlow's Metric Callback
```
Comment (psfoley) [inline docs/log_metric_callback.rst]: I would suggest removing the `a`:

Example of MLFlow's Metric Callback

File: docs/log_metric_callback.rst [hunk 2]
```diff
@@ -104,8 +104,8 @@ Follow the steps below to write your custom callback function instead. As an exa
         template : src.mnist_utils.callback_name
 
 
-Example of a Metric Callback
-============================
+Example of a TensorBoard's Metric Callback
```
Comment (psfoley) [inline docs/log_metric_callback.rst]: I would suggest removing the `a`:

Example of TensorBoard's Metric Callback

**Comment by psfoley** [comment]
Thanks @akantak for the fixes! I left two comments about phrasing. After those are addressed, I think this is ready to merge

**Comment by akantak** [comment]
fixed @psfoley

**Review by psfoley — APPROVED** [review]

**Pull request merged**